python/vehicle-routing/static/app.js

```diff
@@ -20,6 +21,37 @@ const visitGroup = L.layerGroup().addTo(map);
 const homeLocationGroup = L.layerGroup().addTo(map);
 const routeGroup = L.layerGroup().addTo(map);
 
+/************************************ Time line constants and variable definitions ************************************/
+
+const byVehiclePanel = document.getElementById("byVehiclePanel");
```

Comment:
Please make sure this file is identical to the Java frontend.
(I haven't checked whether it is or not.)

Reply:
It was copied from the Java frontend; there is a difference; recommended-fit parts was removed because Python does not support recommended-fit yet.